aptos-move/aptos-vm/src/block_executor/vm_wrapper.rs

```diff
@@ -42,7 +42,10 @@ impl ExecutorTask for AptosExecutorTask {
     // execution, or speculatively as a part of a parallel execution.
     fn execute_transaction(
         &self,
-        view: &(impl ExecutorView + ResourceGroupView + AptosCodeStorage),
+        view: &(impl ExecutorView
```

Comment:
`ExecutorView: BlockSynchronizationKillSwitch` already, I guess we can not extend it and instead pass as is here